css/tutorial.styl

```diff
@@ -25,10 +25,14 @@
     > .media-card-content
       @extends .content-container
 
+    img
+      max-width: 100%
```

Comment:
This isn't necessary. The image is already getting `max-width: 100%` from the `media-card-media` class that is applied to the `<img>` tag.